src/robotide/ui/tagdialogs.py

```diff
@@ -38,7 +38,7 @@ def __init__(self, controller, frame):
         self.unique_tags = 0
         self.total_test_cases = 0
         self.itemDataMap = dict()
-        self.sort_state = (1, 0)
+        self.sort_state = (0, 1)
```

Comment:
This change means, that initial sort order was descending on number of tests by tag, changed to tag names sorted ascending.